docker/k8s-sglang-service.yaml

```diff
@@ -35,10 +35,20 @@ spec:
           env:
             - name: HF_TOKEN
               value: <secret>
+            - name: NUMBA_CACHE_DIR
```

Comment:
I guess they are not necessary ?

Reply:
That's for the other issue, write access in case you do not use root user. numba, outlines and triton are writing some cache so I'm telling it to write to writable /tmp

Reply:
I mean... since it _is_ a different issue, it might got separate PR, but I just assumed once I am updating the manifest, why not add more enhancements than just one...